Vagrantfile

```diff
@@ -1,83 +1,81 @@
 # -*- mode: ruby -*-
 # vi: set ft=ruby :
 
-VAGRANTFILE_API_VERSION = "2"
+# Overwrite host locale in ssh session
+ENV["LC_ALL"] = "en_US.UTF-8"
 
-
-Vagrant.configure(VAGRANTFILE_API_VERSION) do |cluster|
-  # All Vagrant configuration is done here. The most common configuration
-  # options are documented and commented below. For a complete reference,
-  # please see the online documentation at vagrantup.com.
+# All Vagrant configuration is done here.
+Vagrant.configure("2") do |cluster|
+  # The most common configuration options are documented and commented below.
+  # For more refer to https://www.vagrantup.com/docs/vagrantfile/
 
   # Every Vagrant virtual environment requires a box to build off of.
 
-cluster.vm.define "ansible-node" do |config|
-  config.vm.box = "centos/7"
-  config.ssh.insert_key = false
-  config.ssh.forward_agent = true
-  config.vm.provider :virtualbox do |vb, override|
-    vb.customize ["modifyvm", :id, "--memory", "128"]
-    vb.customize ["modifyvm", :id, "--cpus", "1"]
-  end
-  config.vm.hostname = "ansible-node"
-  config.vm.network :private_network, ip: "10.42.0.2"
-end
-
+  # The ordering of these 2 lines expresses a preference for a hypervisor
+  cluster.vm.provider "virtualbox"
+  cluster.vm.provider "vmware_fusion"
```

Comment:
We're not prepared to support VMWare at this time that this line give me pause.

Reply:
Some people have vmware_fusion, and these line make sure that the provisioning avoids using vmware (which has precedence in Vagrant because its plugin is sold). So keep it free (as in beer) with these 2 lines

Reply:
Fair enough. Are there any others we should be looking out for?

Reply:
We should also make sure anyone using Fedora sets virtualbox above libvirt also (boxes).